modules/about-openshift-gitops.adoc

```diff
@@ -0,0 +1,10 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/working-with-openshift-gitops.adoc
+
+[id="about-openshift-gitops_{context}"]
+= About OpenShift GitOps
```

Comment:
s/About OpenShift GitOps/ = About GitOps

There is no difference in About OpenShift GitOps and Overview of OpenShift GitOps. Suggest making the first section - About GitOps and then About Red Hat OpenShift GitOps 

I do not see the include statement for this module in the working-with-openshift-gitops.adoc assembly.